modes/longitudinal/src/toolbarButtons.js

```diff
@@ -423,6 +423,14 @@ const toolbarButtons = [
             },
           ]
         ),
+        // TODO: we need a new icon for image overlay toggler button
+        _createToggleButton('ImageOverlay', 'link', 'Image Overlay', [
+          {
+            commandName: 'toggleImageOverlay',
```

Comment:
Consider a generic toggle operation which can take a tool group to toggle between active and disabled here.